### ISSUE TITLE
"Play" button on manga in library

### ISSUE THREAD

**Issue by rcdailey** [tachiyomiorg/tachiyomi#3071]
### Why/User Benefit/User Problem

As a user, I'd like the ability to more easily and quickly resume a manga from the manga list. The process of resuming a manga today is as follows:

1. Click the manga in the list
2. Navigate to the "Chapters" tab
3. Click the "Play" (resume) button at the bottom right.

While not really inconvenient as it is today, it does make it a little tedious when you're catching up on 5+ different manga that each only have 1 or 2 updated chapters. You actually can spend quite a bit of time just clicking through UI to do a simple task.

### What/Requirements

To simplify the user experience of resuming manga, I suggest adding a "Play" button (similar to the one you see in the Chapters tab for a manga, except without the "Resume" text) on top of the manga image for each manga in the manga list. The button would be small enough that it doesn't interfere with the user's ability to simply click the manga image, which would still take them to the "Detail" view for that manga. Clicking the play button (which should be anchored to the bottom-right corner of each manga entry in the list) will act as if the user clicked the "Resume" button on the Chapters tab for that manga. It simply removes steps 1 and 2 in the steps given earlier.

**Comment by Soitora** [comment]
Which version are you on? By default on the latest I belive you always enter into the Chapters tab as a first-action when clicking a manga in your Library.

**Comment by rcdailey** [comment]
I'm on 0.9.2. If such an option exists, that's a small improvement but unfortunately doesn't completely address the issue. I want to start reading a manga from where I left off directly from the manga list with only one click. I described more details in my OP.

I appreciate your reply!

**Comment by Soitora** [comment]
I see, I'm on the preview but I think that we're not too far off in terms of changes. Have you tried getting used to the History screen? It's basically exactly what you want, a quick Resume button to get you started with reading from where you left.

**Comment by rcdailey** [comment]
Yes I've used the history view, and it's useful for continuing manga that I'm up to speed on, but not really functional for titles I'm still catching up on. I tried to come up with an idea that works for many use cases (following latest chapters, not on latest chapter, reading many manga or just a few).

I feel like my idea provides a more streamlined, convenient, and consistent workflow that just doesn't exist today. Basically today I'm bouncing all over the UI when I shouldn't have to.

Thanks again for the great ideas.
